src/cdk/stack.ts

```diff
@@ -31,6 +31,16 @@ export class MiraServiceStack extends cdk.Stack {
           account: account.env.account
         }
       })
+
+      Tag.add(this, 'StackName', this.stackName)
+      Tag.add(this, 'CreatedBy', account.env.account)
+
+      const costCenter = MiraConfig.getCostCenter()
+
+      if (costCenter) {
```

Comment:
If I don't have a `costCenter` entry then I get 
>Failed: could not deploy the stack Error: Configuration property "costCenter" is not defined

Can you check the behaviour of the config module to make sure it is behaving as expected?